tools/export-docu-to-website.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
@mossmaurice i created an alternative to your bash script in python, because i think we should not introduce more bash script than necessary. Check out if it works for you.

Reply:
Thanks for your work! However, for me a simple Bash script is much more readable than a Python one. Are you planning to run this on more than one platform? Is there a need, that outweighs the larger LOC and poorer readability? Was there a decision we'll re-write all bash scripts to Python? Why do we still have the bash script in the /tools/? ;-)

Reply:
I can remove it if you prefer bash, no problem.

Reply:
No hard feelings on this one. But I'd love to find an answer to our strategy script-wise and the questions raised above.

Reply:
@elfenpiff doesn't like bash scripts because they cannot be used on windows. This one is different though, since it will only be executed on a linux machine. For consistency reasons, I would suggest to not have to many scripting languages. One would be perfect.

Reply:
After talking to @dkroenke we decided to remove the Python script and use it only when we need portability.